pages/docs.html

```diff
@@ -34,13 +34,13 @@ <h1 class="nav-h1">Getting Started</h1>
         <h1 class="nav-h1">Components</h1>
         <ul class="components-box">
             <li class="comp-link"><a href="/pages/components/Avatar-badge/avatar-badge.html">Avatar-Badge</a></li>
-            <li class="comp-link"><a href="#">Alert</a></li>
-            <li class="comp-link"><a href="#">Button</a></li>
-            <li class="comp-link"><a href="#">Cards</a></li>
-            <li class="comp-link"><a href="#">Image-input</a></li>
-            <li class="comp-link"><a href="#">Chips</a></li>
-            <li class="comp-link"><a href="#">Tabs</a></li>
-            <li class="comp-link"><a href="#">Modal</a></li>
+            <li class="comp-link"><a href="/pages/components/Alert/alert.html">Alert</a></li>
+            <li class="comp-link"><a href="/pages/components/Button/button.html">Button</a></li>
+            <li class="comp-link"><a href="/pages/components/Cards/cards.html">Cards</a></li>
+            <li class="comp-link"><a href="/pages/components/Image-input/image-input.html">Image-Input</a></li>
```

Comment:
Better to have Image and Input as separate Components

Reply:
okay will try to change it.

Reply:
i will add outline for alerts

Reply:
Or just use the different background colors. Like primary color, secondary color, etc.